env/mock_env.cc

```diff
@@ -379,7 +379,8 @@ class TestMemLogger : public Logger {
       gettimeofday(&now_tv, nullptr);
       const time_t seconds = now_tv.tv_sec;
       struct tm t;
-      localtime_r(&seconds, &t);
+      auto ret = localtime_r(&seconds, &t);
```

Comment:
You may need to add "__attribute__((unused))" after ret, otherwise release build may complain ret is not used.